src/common/ImageAvatar.js

```diff
@@ -1,32 +1,27 @@
 /* @flow */
 import React from 'react';
-import { Image, StyleSheet } from 'react-native';
+import { Image } from 'react-native';
 
 import { Touchable } from './';
-import UserStatusIndicator from '../common/UserStatusIndicator';
 
-const styles = StyleSheet.create({
-  status: {
-    marginLeft: 21,
-    marginTop: 21,
-  },
-});
+const SHAPE_SQUARE = 'square';
```

Comment:
I don't think adding a constant here necessarily helps.
We will also add some Flow definitions that will help (the type will be 'square' | 'rounded' | 'circle')